controller/tasks_manager.go

```diff
@@ -301,11 +301,11 @@ func (tm TasksManager) addTask(ctx context.Context, d driver.Driver) (config.Tas
 	return conf, nil
 }
 
-func (tm TasksManager) cleanupTask(ctx context.Context, name string) {
-	err := tm.TaskDelete(ctx, name)
-	if err != nil {
-		tm.logger.Error("unable to cleanup task after error", "task_name", name)
-	}
+// cleanupTask cleans up a newly created task that has not yet been added to CTS
+// and started monitoring. Use TaskDelete added and monitored tasks
```

Comment:
nit: I think this was supposed to be "Use TaskDelete for added and monitored tasks"

Reply:
Yes, thank you for catching!